### PR TITLE
Rename some metrics

### PR DESCRIPTION
**Pull request by achamayou**
1. shorten metric names to allow 3 column layout
2. set direction correctly

### CODE REVIEW

File: samples/apps/smallbank/smallbank.cmake [hunk 1]
```diff
@@ -74,7 +74,7 @@ if(BUILD_TESTS)
       NAME small_bank_sigs_forwarding
       PYTHON_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/tests/small_bank_client.py
       CLIENT_BIN ./small_bank_client
-      LABEL Small_Bank_ClientSigs_Forwarding
+      LABEL "SB sig fwd"
```
Comment (jumaffre) [inline samples/apps/smallbank/smallbank.cmake]: Can we avoid spaces? The label is used for the directory name of the test in `workspace/`.

Comment (achamayou) [reply samples/apps/smallbank/smallbank.cmake]: ok

**Comment by cimetrics** [comment]
rename_metrics@[3706 aka 20200107.25](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3706&view=results) vs master ewma over 20 builds from [3604](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3604&view=results) to [3704](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3704&view=results)
images

**Comment by cimetrics** [comment]
rename_metrics@[3708 aka 20200107.26](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3708&view=results) vs master ewma over 20 builds from [3604](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3604&view=results) to [3704](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3704&view=results)
images

**Comment by cimetrics** [comment]
rename_metrics@[3710 aka 20200107.27](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3710&view=results) vs master ewma over 20 builds from [3604](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3604&view=results) to [3704](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3704&view=results)
images

**Comment by cimetrics** [comment]
rename_metrics@[3716 aka 20200107.30](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3716&view=results) vs master ewma over 20 builds from [3605](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3605&view=results) to [3714](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3714&view=results)
images

**Pull request merged**